.gitignore

```diff
@@ -1 +1,3 @@
 node_modules/
+.DS_Store
+package-lock.json
```

Comment:
Hey, 
Not sure about adding the package-lock.json to our .gitignore. Some of the reading I've done leads me to believe that it should remain in there. 
https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore
https://github.com/npm/cli/blob/release-6.14.7/docs/content/configuring-npm/package-lock-json.md
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Let me know what you think, or if I've got this wrong.

Reply:
It's meant to be added to version control but since they added it to npm 5.0, it has caused more headaches for me than benefits. There are comments in the articles that have had similar experiences. It's not a big deal, I'll just get used to it